### PR TITLE
[ADD] l10n_es_edi_facturae_invoice_period: invoice periods for EDI

### PR DESCRIPTION
**Pull request by alialfie**
Patch module to add missing fields to the factura-e xml.

task-3673113

### CODE REVIEW

**Review by william-andre — COMMENTED** [review]
Code looks good :+1: 

Need to add the translations though.

Would also be nice if we were sure of what we need, there was already a patch module for ADM centers...

**Comment by alialfie** [comment]
@william-andre Thanks for the review! Added the translations and double checked with Christophe and we're sure we need this at the moment

File: addons/l10n_es_edi_facturae_invoice_period/i18n/l10n_es_edi_facturae_invoice_period.pot
Comment (william-andre) [inline addons/l10n_es_edi_facturae_invoice_period/i18n/l10n_es_edi_facturae_invoice_period.pot:1]: What we actually need is the translation in Spanish, as this is not on Transifex

**Review by william-andre — COMMENTED** [review]
> double checked with Christophe and we're sure we need this at the moment

Yes, I wasn't implying that this would not be useful, I was just hoping that this was the last bridge needed and that no other similar feature would come in the next days/weeks :smile:

**Comment by alialfie** [comment]
> > double checked with Christophe and we're sure we need this at the moment
> 
> Yes, I wasn't implying that this would not be useful, I was just hoping that this was the last bridge needed and that no other similar feature would come in the next days/weeks 😄

@chklop :arrow_up:

File: addons/l10n_es_edi_facturae_invoice_period/i18n/l10n_es_edi_facturae_invoice_period.pot
Comment (alialfie) [reply addons/l10n_es_edi_facturae_invoice_period/i18n/l10n_es_edi_facturae_invoice_period.pot:1]: Wasn't aware of that. Updated it!

File: addons/l10n_es_edi_facturae_invoice_period/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#: model:ir.model.fields,field_description:l10n_es_edi_facturae_invoice_period.field_account_bank_statement_line__l10n_es_invoicing_period_start_date
+#: model:ir.model.fields,field_description:l10n_es_edi_facturae_invoice_period.field_account_move__l10n_es_invoicing_period_start_date
+#: model:ir.model.fields,field_description:l10n_es_edi_facturae_invoice_period.field_account_payment__l10n_es_invoicing_period_start_date
+msgid "Invoice Period Start Date"
+msgstr "Fecha de inicio del periodo de facturación"
+
+#. module: l10n_es_edi_facturae_invoice_period
+#: model:ir.model,name:l10n_es_edi_facturae_invoice_period.model_account_move
+msgid "Journal Entry"
+msgstr "Anotación en el diario"
```
Comment (william-andre) [inline addons/l10n_es_edi_facturae_invoice_period/i18n/es.po:41]:
```suggestion
msgstr "Asiento contable"
```

Comment (william-andre) [reply addons/l10n_es_edi_facturae_invoice_period/i18n/es.po:41]: From other translations in other modules

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**